liger_iris_pipeline/tests/test_rop.py

```diff
@@ -60,8 +60,9 @@ def setup_inputs(
 
 
 def test_rop1():
+    raw_readout_filename = get_data_from_url("48191977")
     liger_iris_pipeline.pipeline.ROPPipeline.call(
-        "liger_iris_pipeline/tests/data/test_ramp.fits", config_file="liger_iris_pipeline/tests/data/drsrop.cfg"
```

Comment:
how did you create this new `test_ramp.fits`? is it worth saving the script used to create it?
or otherwise if it is simple enough, can we generate it on the fly instead of getting it from figshare?

Reply:
I didn't create new ramps from scratch. After tweaking the header I uploaded [data/test_ramp.fits](https://github.com/oirlab/liger_iris_pipeline/blob/master/liger_iris_pipeline/tests/data/test_ramp.fits) to [figshare](https://figshare.com/articles/dataset/IRIS_Liger_DRS_Test_Data/26492029?file=48191977). I saw it's a tiny file but I preferred the testing .fits files in one place.

Reply:
ok